src/util/ical.ts

```diff
@@ -103,6 +117,46 @@ function icalEventToTask(
   };
 }
 
+// Helper function to determine RSVP status
+function getRsvpStatus(icalEvent: WithIcalConfig<ical.VEvent>): string {
```

Comment:
Ical events have a `status` field (at least the ones from Outlook that I've just checked):
![image](https://github.com/user-attachments/assets/b86f9ecf-cda2-43b1-b37d-93e255b5abb6)
Have you checked them on Google events?

If this is so, you can get rid of all this parsing all together and just use status straight from the event.